package.json

```
@@ -67,6 +67,11 @@
                 "runOnSave": {
                     "type": "boolean",
                     "default": true
+                },
+                "insertCommaAfterEnter": {
```

```suggestion
                "insertMissingCommaOnEnter": {
```
As I remember in WS the feature is called `Insert missing commas on Enter` let's call it in the same way